lib/routes/route.js

```diff
@@ -1061,22 +996,27 @@ module.exports = function(app, router) {
 
         databaseOps.addUser(userData, function(err, result) {
             if (err) {
-                var field;
+                let field;
+                let statusCode = 500;
+                let errorID = "databaseError";
                 if (err.message !== undefined) {
                     errMsg = err.message;
```

Comment:
Where is `errMsg` defined locally and what happens if there's no `err.message` but no duplicate username either?